setup_dev_env.sh

```diff
@@ -36,11 +36,25 @@ fi
 # define a python version to initialise the conda environment.
 # by default we assume python 3.7.
 python_version=${PYTHON_VERSION:-"3.7"}
+pytorch_version=${PYTORCH_VERSION:-"1.6.0"}
+pytorch_mode=${PYTORCH_MODE:-""}  # use `cpuonly` for CPU or leave it in blank for GPU
 
 # create an environment with the specific python version
 $conda_bin config --append channels conda-forge
-$conda_bin config --append channels pytorch
 $conda_bin update -n base -c defaults conda
-$conda_bin create --name venv --file requirements.txt python=$python_version
-
+$conda_bin create --name venv python=$python_version
 $conda_bin clean -ya
+
+# activate local virtual environment
+source $conda_bin_dir/activate $dev_env_dir/envs/venv
+
+# install pytorch and torchvision
+conda install pytorch=$pytorch_version torchvision $pytorch_mode -c pytorch
+
+# install testing dependencies
+pip install -r requirements-dev.txt
+
+# install documentation dependencies
+pip install -r docs/requirements.txt
```

Comment:
Forgot adding in the file?

Reply:
Nope, this file has existed for a while used only by read the docs